code/modules/lighting/lighting_overlay.dm

```diff
@@ -3,7 +3,6 @@
 	mouse_opacity = 0
 	simulated = 0
 	anchored = 1
-	flags = NOREACT
```

Comment:
Probably to avoid someone somehow managing to get blackpowder and phlog into a lighting overlay and blowing up light itself?